projects/extension/src/background/AppMediator.ts

```diff
@@ -21,9 +21,9 @@ import polkadot from '../../public/assets/polkadot.json';
 type RelayType = Map<string, string>;
 
 export const relayChains: RelayType = new Map<string, string>([
-  ["polkadot", JSON.stringify(polkadot)],
-  ["kusama", JSON.stringify(kusama)],
-  ["westend", JSON.stringify(westend)]
+  ['polkadot', JSON.stringify(polkadot)],
+  [kusama.id, JSON.stringify(kusama)],
+  [westend.id, JSON.stringify(westend)]
```

Comment:
```suggestion
  ["polkadot", JSON.stringify(polkadot)],
  ["kusama.id", JSON.stringify(kusama)],
  ["westend.id", JSON.stringify(westend)]
```

These strings are part of the API of substrate-connect. The values found in the chain spec are an implementation detail.